templates/pages/comment_edits_history.html

```diff
@@ -19,27 +20,46 @@
     <li>{% trans "Historique des éditions" %}</li>
 {% endblock %}
 
-
-
 {% block headline %}
     <h1>{% trans "Historique des éditions du message" %}</h1>
 {% endblock %}
 
-
-
 {% block content_page %}
     {% if edits %}
         <table class="fullwidth">
             <thead>
+                <th></th>
                 <th class="wide">{% trans "Date" %}</th>
                 <th>{% trans "Éditeur" %}</th>
-                <th>{% trans "Version avant édition" %}</th>
+                <th>{% trans "Action" %}</th>
             </thead>
             <tbody>
+                {% sibling edits 0 as current %}
+                <tr>
+                    <td>
+                        Edition {{ edits|length }}
```

Comment:
```suggestion
                        Édition {{ edits|length }}
```

Accent sur la majuscule (`Édition`).